MediaBrowser.Controller/Net/BasePeriodicWebSocketListener.cs

```diff
@@ -98,19 +90,10 @@ private void Start(WebSocketMessageInfo message)
             var dueTimeMs = long.Parse(vals[0], UsCulture);
             var periodMs = long.Parse(vals[1], UsCulture);
 
-            if (vals.Length > 2)
-            {
-                ParseMessageParams(vals.Skip(2).ToArray());
```

Comment:
what was this for?

Reply:
Dunno, but `ParseMessageParams` was never overridden, so I removed it.